.github/workflows/tests.yml

```diff
@@ -337,7 +337,7 @@ jobs:
             dev/utils.ts
           deno --unstable-worker-options --unstable-net coverage ./coverage --lcov > coverage.lcov
 
-      - uses: codecov/codecov-action@v3
+      - uses: codecov/codecov-action@v4
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
Looks like the new changes with codecov require the token to be passed through the CODECOV_TOKEN env var. Also, setup-ghjk is managed through `dev/lock.ts` thus why `test-pre-commit` is failing. Will fix.

Reply:
@zifeo turns out the old `token: <token here>` syntax is still valid. Is it possible `secret.CODECOV_TOKEN` isn't set on this repository? (it used to work tokenless prior to v4 according to their docs)

Reply:
ah, nvm. Seems to be working now.